src/style-installer.js

```diff
@@ -131,6 +131,7 @@ export function installStylesLegacy(
  * @return {!Element}
  */
 function insertStyleElement(cssRoot, cssText, isRuntimeCss, ext) {
+  cssText = cssText.trim();
```

Comment:
Do we need to trim?
Because if we do we should make it so that we don't have to.

Reply:
We should consider that invalid for SSR. @honeybadgerdontcare 

Let's avoid the trimming here. It may increase memory usage.

Reply:
Removed the trimming.

Reply:
Agreed, we can assist with pub SSR generation. Filed b/112736946 and b/112737536.